### PR TITLE
Add meshnet grpc-wire option

### PR DESCRIPTION
**Pull request by alexmasi**

### CODE REVIEW

File: manifests/meshnet/grpc/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        - name: INTER_NODE_LINK_TYPE
+          value: GRPC
+        image: us-west1-docker.pkg.dev/kne-external/kne/networkop/meshnet:v0.3.1
+        imagePullPolicy: IfNotPresent
+        name: meshnet
+        resources:
+          limits:
+            memory: 1000Mi
+          requests:
+            cpu: 100m
```
Comment (guoshiuan) [inline manifests/meshnet/grpc/manifest.yaml:164]: Just curious... what does 100m mean for CPU?

File: manifests/meshnet/vxlan/manifest.yaml [hunk 2]
```diff
@@ -170,9 +177,9 @@ spec:
       hostNetwork: true
       hostPID: true
       nodeSelector:
-        kubernetes.io/arch: amd64
+        beta.kubernetes.io/arch: amd64
```
[the remark this answers is not in the document]
Comment (alexmasi) [reply manifests/meshnet/vxlan/manifest.yaml:180]: not sure, this is a sync from meshnet and should be treated as an ingestion

File: manifests/meshnet/grpc/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+            properties:
+              links:
+                items:
+                  description: A complete definition of a p2p link
+                  properties:
+                    local_intf:
+                      description: Local interface name
+                      type: string
+                    local_ip:
+                      description: (Optional) Peer IP address
```
[the remark this answers is not in the document]
Comment (alexmasi) [reply manifests/meshnet/grpc/manifest.yaml:38]: done

File: manifests/meshnet/grpc/manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+                      type: string
+                    peer_intf:
+                      description: Peer interface name
+                      type: string
+                    peer_ip:
+                      description: (Optional) Local IP address
```
[the remark this answers is not in the document]
Comment (alexmasi) [reply manifests/meshnet/grpc/manifest.yaml:44]: done

File: manifests/meshnet/grpc/manifest.yaml (hunk 1 above)
Comment (alexmasi) [reply manifests/meshnet/grpc/manifest.yaml:164]: 100 milliCPU

https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit

File: manifests/meshnet/vxlan/manifest.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexmasi) [reply manifests/meshnet/vxlan/manifest.yaml:180]: https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit

**Review by guoshiuan — APPROVED** [review]

**Pull request merged**